### PR TITLE
Fixed wrong field name for accountid

### LINKED ISSUE #2: Outdated field name
I think that the script that is creating a default configuration file has an outdated field name. According to Maxmind instructions (https://dev.maxmind.com/geoip/geoipupdate/) we should mention an **AccountID**, yet on https://github.com/whitfin/docker-geoipupdate/blob/master/src/exec.sh#L7 an **UserId** is added.

### PR DESCRIPTION
**Pull request by nhmarujo**
Fixes #2

### CODE REVIEW

**Review by halkeye — APPROVED** [review]

File: src/exec.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@
 ! (: "${GEOIP_CONFIG_FILE?}") 2>/dev/null && {
     GEOIP_CONFIG_FILE="/etc/GeoIP.conf"
     [[ ! -z $GEOIP_USER_ID ]] && {
```
Comment (halkeye) [inline src/exec.sh:6]:
```suggestion
     [[ ! -z $GEOIP_ACCOUNT_ID ]] && {
```

**Comment by whitfin** [comment]
I don't think the `UserId` field is "wrong", because it's functioning fine in a build system at work. 

So, to fix this, I think we should copy the logic [here](https://github.com/whitfin/docker-geoipupdate/blob/master/src/exec.sh#L6-L8) and place it below except for `AccountId` accordingly. That way people can use both (i.e. we don't break existing usage).

**Comment by halkeye** [comment]
Maybe it's different versions of the codebase? Or maybe newer licenses. I just created an account last night and the module complained about account id. I just switched to the file they gave me instead of the env vars

**Comment by whitfin** [comment]
Yeah, I assume they switched to a new licensing system which uses `AccountId` or something and left the old for backwards compatibility. I just re-ran our builds and they 100% work with `UserId` though, so I definitely want to keep both to avoid having to force people to switch.

**Comment by bgli100** [comment]
It seems MaxMind now requires `AccountId` and no longer accepts `UserId`?

**Comment by whitfin** [comment]
Hi all, 

Really sorry that this got lost in my inbox, I guess I can only blame covid and then a bad memory 😞 

I have updated the images with support for the new variables (they also changed ProductIds) and I have pushed all versions that were missing through to the latest. 

I'm also going to set up Actions to make sure I don't miss these in future!

**Pull request closed without merging**